tutorials/cosmwasm.md

```diff
@@ -127,9 +123,22 @@ Cosmos-SDK applications to connect to Celestia's data availability network.
 git clone https://github.com/CosmWasm/wasmd.git
 cd wasmd
 git checkout tags/v0.50.0
-go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/rollkit/cosmos-sdk@v0.50.1-rollkit-v0.11.9-no-fraud-proofs
+go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/rollkit/cosmos-sdk@v0.50.1-rollkit-v0.11.19-no-fraud-proofs
 go mod tidy -compat=1.17
 go mod download
+```
+
+Now, comment out lines 902-904 in `app/app.go`:
```

Comment:
do you want to leave a note here saying that this is a temporary hack until https://github.com/CosmWasm/wasmd/issues/1785 is resolved?

Reply:
good call, yes, adding now f4384b7afcca619738fe43ca758e8df54f46a16c